rest_framework/throttling.py

```diff
@@ -101,9 +102,26 @@ def parse_rate(self, rate):
         """
         if rate is None:
             return (None, None)
-        num, period = rate.split('/')
-        num_requests = int(num)
-        duration = {'s': 1, 'm': 60, 'h': 3600, 'd': 86400}[period[0]]
+
+        try:
+            num, period = rate.split('/')
+            num_requests = int(num)
+            period_timescale = re.findall(r'^\d*[s|m|h|d]', period)[0]
+            period_char = period_timescale[-1]
+        except (ValueError, IndexError):
+            msg = "Incorrect throttle rate set for '%s' scope" % self.scope
+            raise ImproperlyConfigured(msg)
+
+        try:
+            # Get rate multiplier value if available
+            period_mult = int(period_timescale[:-1])
+        except ValueError:
```

Comment:
This handles the case where multiplier is not provided. The rate validation is already performed at this point, so we're only expecting valid strings like `10h` and `h` here.